docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1108,99 +1138,100 @@ Enables [pruning](../../Concepts/Pruning.md) to reduce storage required for the
 !!! important
     Using pruning with [private transactions](../../Concepts/Privacy/Privacy-Overview.md) is not supported.
```

Comment:
The next three options after `--pruning-enabled` are indented inside the important box. If you publish locally, you'll be able to see what I mean.

Reply:
Fixed.